Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -211,56 +226,51 @@ extension GeometryEditorMenu {
     }
 }
 
-extension GeometryEditorMenu {
+private extension GeometryEditorMenu {
     /// A Boolean value indicating whether the selection can be deleted.
     ///
     /// In some instances deleting the selection may be invalid. One example would be the mid vertex
     /// of a line.
-    var deleteButtonIsDisabled: Bool {
-        guard let selection = model.selection else { return true }
-        return !selection.canBeDeleted
+    private var deleteButtonIsDisabled: Bool {
```

Comment:
Since the extension itself is private, annotating the members as private is superfluous.

```suggestion
    var deleteButtonIsDisabled: Bool {
```